File: generated-sources/csharp-dotnet2/mojang-api/src/main/CsharpDotNet2/Org/OpenAPITools/Model/SecurityAnswerId.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// A security question answer id given by Mojang
  /// </summary>
  [DataContract]
  public class SecurityAnswerId {
    /// <summary>
    /// A number giver by Mojang to reply a question
    /// </summary>
    /// <value>A number giver by Mojang to reply a question</value>
    [DataMember(Name="id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "id")]
    public int? Id { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SecurityAnswerId {\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: generated-sources/dart/mojang-api/com/github/asyncmc/mojang/api/dart/model/security_answer_id.dart
part of openapi.api;

class SecurityAnswerId {
  /* A number giver by Mojang to reply a question */
  int id = null;
  SecurityAnswerId();

  @override
  String toString() {
    return 'SecurityAnswerId[id=$id, ]';
  }

  SecurityAnswerId.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    id = json['id'];
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id
    };
  }

  static List<SecurityAnswerId> listFromJson(List<dynamic> json) {
    return json == null ? new List<SecurityAnswerId>() : json.map((value) => new SecurityAnswerId.fromJson(value)).toList();
  }

  static Map<String, SecurityAnswerId> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, SecurityAnswerId>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new SecurityAnswerId.fromJson(value));
    }
    return map;
  }
}

